packages/service-library/src/web-api/scan-notification-error-codes.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// tslint:disable: no-unnecessary-class variable-name
+
+export declare type ScanNotificationErrorCodeName = 'InternalError' | 'HttpErrorCode';
+
+export interface ScanNotificationErrorCode {
+    // This type is part of the REST API client response.
+    // Ensure compatibility when changing this type.
+    code: ScanNotificationErrorCodeName;
+    codeId: number;
+    message: string;
+}
+
+export class ScanNotificationErrorCodes {
+    public static internalError: ScanNotificationErrorCode = {
+        code: 'InternalError',
+        codeId: 9001,
+        message: 'The scan engine encountered an internal error.',
+    };
+
+    public static HttpErrorCode: ScanNotificationErrorCode = {
+        code: 'HttpErrorCode',
+        codeId: 9012,
+        message: 'Failed to send scan notification.',
```

Comment:
this is too generic error message, not clear how this correlates with Http... should code be just a `NotificationError` instead?